src/applications/claims-status/tests/e2e/01.claim-status-decision.e2e.spec.js

```diff
@@ -18,10 +18,10 @@ module.exports = E2eHelpers.createE2eTest(client => {
     .waitForElementVisible('body', Timeouts.normal)
     .waitForElementVisible('.claim-title', Timeouts.slow);
 
-  client.expect.element('.usa-alert').to.be.visible;
+  client.expect.element('.main .usa-alert').to.be.visible;
```

Comment:
The global banner was causing issues with our selectors in our E2E tests, so instead of doing something hacky like removing the banner during E2E via a build flag, I used more specific selectors.